sparse_strips/vello_common/src/encode.rs

```diff
@@ -394,6 +394,10 @@ fn encode_stops(
     // Create additional (SRGB-encoded) stops in-between to approximate the color space we want to
     // interpolate in.
     if cs != ColorSpaceTag::Srgb {
+        // TODO: The `color` crate will always interpolate in premultiplied space, while for Srgb
+        // we do it in unpremultiplied space since this is more desirable for most 2D rendering
+        // situations (see https://github.com/linebender/vello/issues/1129), so the results might
+        // look off until this is implemented in the `color` crate.
```

Comment:
Is this [the issue](https://github.com/linebender/color/issues/184) in `color` crate related to what you mentioned? Can we link it here, even though it will hopefully be closed soon?